### PR TITLE
Vitefy

### LINKED ISSUE #8: Need For Speed
Add a script that will allow users to migrate this react app to one that is served by [vite](https://vitejs.dev/) The script should perform the following:
- [ ] Rename all .js files to .jsx
- [ ] Delete the index.html file and create another one in the root folder. It should contain a script that loads the ```index.jsx``` file as a module.
- [ ] Create a new ```vite.config.js``` file at the root of the project
- [ ] Install ```vite``` and ```@vitejs/plugin-react``` as dev dependencies.
- [ ] Replace the ```start``` and ```build``` scripts in package.json to ```vite```  and ```vite build --outDir build``` respectively.
- [ ] Replace all env variables in the env files to prefix with ```VITE_``` instead of ```REACT_APP_``` (Optional)
- [ ] Prints out a statement telling the user to change all the ```process.env``` to ```import.meta.env```. See the [docs](https://vitejs.dev/guide/env-and-mode.html) for more info.
- [ ] Right now the app uses npm for installing packages. But for security and possible migrations to yarn, the script should decide as to whether to install ```vite``` via ```yarn``` or ```npm```.
- [ ] Finally the user might have already converted the app to a PWA. So script should also prompt the user if he wants a pwa version of this migration. See [vite-plugin-pwa](https://vite-plugin-pwa.netlify.app/guide/#configuring-vite-plugin-pwa) for more info.

Refer to this [article](https://dev.to/nilanth/use-vite-for-react-apps-instead-of-cra-3pkg) for more info

### PR DESCRIPTION
**Pull request by eigengravy**
Closes #8 

`npm run vitefy`

### CODE REVIEW

**Comment by ananth243** [comment]
The goal is to create a command as such "npm run vitefy" and via code it does all of the changes that you have made here. So revert the state of the app to before vitefy.

File: scripts/vitefy.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+          }
+        );
+        unlink(join('public', 'index.html'), err => {
+          if (err) throw err;
+        });
+      });
+    }
+  });
+
+  appendFile(
```
Comment (ananth243) [inline scripts/vitefy.js]: writeFile would do a better job here

**Review by ananth243 — CHANGES_REQUESTED** [review]
Add .env to .gitignore and you're good to go!

**Comment by eigengravy** [comment]
Done @ananth243

**Comment by ananth243** [comment]
@riskycase looks good to me. Check if you have any.

**Pull request merged**